docs/_future/concepts.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Slack developer beta
+lang: en
+slug: concepts
+order: 0
+layout: future
+permalink: /future/concepts
+---
+# Slack developer beta concepts <span class="label-beta">BETA</span>
```

Comment:
Noticed this odd alignment with the top portion of the Concepts page:
<img width="604" alt="Screen Shot 2022-11-15 at 4 21 35 PM" src="https://user-images.githubusercontent.com/12901850/202027980-376f32b2-9fb7-415b-b5cf-ffae14ab634a.png">

this can always be a fast follow to this PR, but it does have some awkward spacing so might be worth taking a look!

Reply:
I feel like this is a css problem with the alert paragraph, not sure where this is defined, I was not able to find it in the styles.css

Reply:
@hello-ashleyintech might be simpler to create a ticket for this and solve it separately not sure how long this could take

Reply:
Sounds good! I created a separate ticket for it in JIRA under the Bolt epic. :tada: